packages/base-driver/lib/protocol/protocol.js

```diff
@@ -239,9 +239,17 @@ function buildHandler (app, method, path, spec, driver, isSessCmd) {
       // this command should never be proxied (which is useful for plugin developers who add
       // commands and generally would not want that command to be proxied instead of handled by the
       // plugin)
+      let pluginOverrodeProxy = false;
```

Comment:
I usually prefix boolean vars with `did` or `is`

Reply:
👍